sdk/FunctionMetadataLoaderExtension/Startup.cs

```diff
@@ -48,9 +49,15 @@ public void Configure(WebJobsBuilderContext context, IWebJobsConfigurationBuilde
         public void Configure(WebJobsBuilderContext context, IWebJobsBuilder builder)
         {
             string appRootPath = context.ApplicationRootPath;
-            builder.Services.AddOptions<FunctionMetadataJsonReaderOptions>().Configure(o => o.FunctionMetadataFileDrectory = appRootPath);
-            builder.Services.AddSingleton<FunctionMetadataJsonReader>();
-            builder.Services.AddSingleton<IFunctionProvider, JsonFunctionProvider>();
+            WorkerConfigDescription newWorkerDescription = GetUpdatedWorkerDescription(appRootPath);
+
+            if (string.IsNullOrEmpty(newWorkerDescription.EnableWorkerIndexing) ||
```

Comment:
This was something I was working on before the new design change (workerIndexing set to true to handle request messages but not necessarily the indexing task) but has no impact on the way indexing would work for Stein b/c we are hardcoding the worker.config file's `workerIndexing` to true.

I did test it w/ `workerIndexing` set to false or not set at all to make sure we could preserve the current function loading behavior w/ the extension if we wanted to.